command/main.go

```diff
@@ -126,8 +131,8 @@ func RunCustom(args []string, runOpts *RunOptions) int {
 	}
 
 	uiErrWriter := runOpts.Stderr
-	if outputCurlString {
-		uiErrWriter = ioutil.Discard
+	if outputCurlString || outputPolicy {
+		uiErrWriter = &bytes.Buffer{}
```

Comment:
Would changing it from `Discard` to an empty buffer not change the behavior?

Reply:
The uiErrWriter gets passed into the actual commands, as the ErrorWriter for the VaultUI object on the next line. If an error is encountered during the cli.Run() step but before LastOutputPolicyError gets populated (such as the error encountered by KV commands' preflight request), previously it would just completely discard the error. 

Changing it to a buffer which we write those errors into is how I surface them again later in generatePolicy, as Tom's comment requested.